README.md

```diff
@@ -6,9 +6,88 @@
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-stripe.svg)](https://github.com/ballerina-platform/module-ballerinax-stripe/commits/master)
 [![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/stripe.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%stripe)
 
-Stripe offers payment processing software and application programming interfaces for e-commerce websites and mobile applications.
+[Stripe](https://stripe.com/) is a leading online payment processing platform that simplifies the handling of financial transactions over the Internet. Stripe is renowned for its ease of integration, comprehensive documentation, and robust API that supports a wide range of payment operations including credit card transactions, subscription management, and direct payouts to user bank accounts.
 
-The Stripe API is organized around REST. Our API has predictable resource-oriented URLs, accepts form-encoded request bodies, returns JSON-encoded responses, and uses standard HTTP response codes, authentication, and verbs.
+The Ballerina Stripe Connector allows developers to interact with the [Stripe REST API](https://stripe.com/docs/api), enabling seamless integration of Stripe’s extensive payment processing capabilities into Ballerina applications. This connector facilitates the automation of various payment-related operations such as charge creation, customer management, billing, and direct payouts. By leveraging the Ballerina Stripe Connector, developers can build secure and scalable payment solutions that enhance the e-commerce capabilities of their applications.
```

Comment:
Let's mention the supported REST API version

Reply:
Still better to mention the REST API version to be future proof IMO (e.g. incase they introduce a new API version in near future)

Reply:
Fixed with [6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/3/commits/6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf)